src/test/java/com/reminiscence/reminiscence/journal/testEntryGetters.java

```diff
@@ -0,0 +1,116 @@
+package com.reminiscence.reminiscence.journal;
+
+import com.reminiscence.reminiscence.account.AccountRepo;
+import com.reminiscence.reminiscence.account.UserAccount;
+import com.reminiscence.reminiscence.journal.Entry;
+import com.reminiscence.reminiscence.journal.EntryRepo;
+import com.reminiscence.reminiscence.watson.ToneRepo;
+import com.reminiscence.reminiscence.watson.WatsonController;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.test.web.servlet.setup.SecurityMockMvcConfigurers;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+
+import static org.junit.Assert.*;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class testEntryGetters {
```

Comment:
Java classes should be PascalCase.  (Capitalize that first letter!)

From what I've seen, test naming conventions are:  `ClassNameTest` ('Test' appended to whatever the subject of the test is)